dev-packages/node-integration-tests/suites/tracing-experimental/prisma-orm/setup.ts

```diff
@@ -0,0 +1,16 @@
+import { execSync } from 'child_process';
+import { parseSemver } from '@sentry/utils';
+
+const NODE_VERSION = parseSemver(process.versions.node);
+
+if (NODE_VERSION.major && NODE_VERSION.major < 16) {
+  // eslint-disable-next-line no-console
+  console.warn(`Skipping Prisma tests on Node: ${NODE_VERSION.major}`);
+  process.exit(0);
+}
+
```

Comment:
Does prisma only support Node 16+? (If so, can we add a comment that this is the reason why we skip?

Reply:
Yes, I just added a comment with the link 👍

https://www.prisma.io/docs/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-5#nodejs-minimum-version-change